README.md

```diff
@@ -3,3 +3,28 @@
 ## Building the plugin
 To build the calico-kubernetes plugin, clone this repository and run `make`.  This will build the binary, as well as run the unit tests.  To just build the binary, with no tests, run `make binary`.  To only run the unit tests, simply run `make ut`.
 
+## Using rkt Plugins
+The [documentation](https://github.com/coreos/rkt/blob/master/Documentation/networking.md) for Network Plugins in rkt will walk you through the basics of setting up networking in rkt.
+
+## Requirements
+* A working [etcd](https://github.com/coreos/etcd) service
+* Currently, Calico is capable of networking rkt containers, but our `calico/node` image does not yet work in rkt. For now, you will need to run our `calico/node` [docker image](https://github.com/projectcalico/calico-docker/blob/master/docs/getting-started/default-networking/Demonstration.md) in parallel to your rkt images.
```

Comment:
I think saying `our calico/node image does not yet work in rkt` is unclear to someone not familiar with the calico packages. How about: 

>Though Calico is capable of networking rkt containers, our core software is distributed and deployed in a docker container. For now, you will need to run calico in docker before starting your rkt contianers. This can be easily done by downloading the calicoctl binary(link) and running the following command: `calicoctl node --ip=$IP`